.github/workflows/artifacts.yaml

```diff
@@ -75,6 +75,9 @@ jobs:
         path: ${{ env.x86_64_image_path }}
   export-docker-image-arm64:
     runs-on: arm-ubuntu-arm-22.04-4core
+    container:
+      image: arm64v8/ubuntu:22.04  # Specify an arm64 Docker image here
+      options: --security-opt=no-new-privileges  # Optional: Add additional Docker security options
```

Comment:
Looks like it fails the presubmit check, could you check it?

Reply:
I skipped this on purpose since the job is run in `runs-on: ubuntu-22.04`, which is a Github-hosted runner. It will have it's own isolation so this is not really necessary.

Reply:
Regardless of the purpose of this PR, it shouldn't break any presubmit checks. This PR is currently breaking `export-docker-image-arm64`.
https://github.com/google/android-cuttlefish/actions/runs/11336098940/job/31525430431?pr=749

Reply:
Sorry, it takes some time to see if it possible to do docker-in-docker for bazel to work on building docker images